src/components/World/World.js

```diff
@@ -22,7 +25,7 @@ export default createPureComponent({
     const { col, row, children } = this.props;
     const style = gridCoordsToOffsetStyle(row, col);
     return (
-      <div className="world" style={style}>
+      <div className={classNames('world', { twemoji: !hasEmoji })} style={style}>
```

Comment:
Also for :corn:sistency, can we create a `className` `const` and use the object syntax—even for `world`—like [this](https://github.com/stevenhauser/i-have-to-return-some-videotapes/blob/master/src/components/EditorTool/EditorTool.js#L32-L35).